web/src/compose_ui.js

```diff
@@ -495,6 +527,50 @@ export function format_text($textarea, type, inserted_content) {
 
             wrapSelection(field, italic_syntax);
             break;
+        case "bulleted": {
+            // Following behaviour is inspired from GitHub
```

Comment:
Which specific behavior -- the toggling if there's already bullets?

Reply:
I actually meant all of it, nothing specific -- the toggling of complete lines even when they are partially selected, the toggling when there's already some bullets and the adding of a blank line before and after the bulleted selection.

Reply:
OK -- I would replace this with just briefly describing the behavior we are implemented -- GitHub's behavior may change over time, and we're not exactly replicating every detail of it (At least I remember finding at least one bug that they have that we don't!), and it's nice to just have explanation be self-contained (or link to a reference, if say we were implementing some stnadard).

Reply:
Makes a lot of sense, have elaborated specifics in the comment